dnf/cli/option_parser.py

```diff
@@ -270,11 +270,11 @@ def _add_general_options(self):
                                 help=_('enable just specific repositories by an id or a glob, '
                                        'can be specified multiple times'))
         enable_group = general_grp.add_mutually_exclusive_group()
-        enable_group.add_argument("--enable", "--set-enabled", default=False,
+        enable_group.add_argument("--enable", default=False,
                                   dest="set_enabled", action="store_true",
                                   help=_("enable repos with config-manager "
                                          "command (automatically saves)"))
-        enable_group.add_argument("--disable", "--set-disabled", default=False,
+        enable_group.add_argument("--disable", default=False,
```

Comment:
We should make sure that dnf and dnf-plugins-core are both updated. Otherwise we can end in situation that `--set-enabled` is removed from global options (due to applying this patch) but is not added to config-managers options (because https://github.com/rpm-software-management/dnf-plugins-core/pull/396 is not forced).

So I think we need to bump versions of both dnf and dnf-plugins-core and change  conflicts_dnf_plugins_core_version in dnf.spec and dnf_lowest_compatible in dnf-plugins-core.spec accordingly.

Reply:
@m-blaha Thanks for the reminder. Done.